ui/app/AppLayouts/Communities/panels/SharedAddressesAccountSelector.qml

```diff
@@ -143,12 +140,20 @@ StatusListView {
                 icon.color: hovered ? Theme.palette.primaryColor3 :
                                       checked ? Theme.palette.primaryColor1 : disabledTextColor
                 checkable: true
-                checked: listItem.address === root.selectedAirdropAddress.toLowerCase()
-                enabled: shareAddressCheckbox.checked && root.selectedSharedAddresses.length > 1 // last cannot be unchecked
+                checked: {
+                    let obj = root.selectedSharedAddressesMap.get(listItem.address)
```

Comment:
Isn't the `address` property also part of the model item???

Reply:
Yes it is.